src/lxc/monitor.c

```diff
@@ -205,7 +204,7 @@ int lxc_monitor_open(const char *lxcpath)
 	}
 
 	len = strlen(&addr.sun_path[1]) + 1;
```

Comment:
With the removal of the 0-termination in `lxc_monitor_sock_name` this `strlen` potentially reads past the `addr` on the stack giving weird results.
(Btw. - I think the length check could be moved above the `socket()` call?)